### ISSUE TITLE
enableRule0011ForTableFields not working?

### ISSUE THREAD

**Issue by tscottjendev** [StefanMaron/BusinessCentral.LinterCop#253]
I have added the `enableRule0011ForTableFields` to may LinterCop.json (inside a project folder or a multi-root workspace) and I am not seeing any warnings on fields.  I only see the warning for the object.  Am I doing something wrong?

LinterCop.json
```
{
    "cyclomaticComplexetyThreshold": 20,
    "maintainablityIndexThreshold": 20,
    "enableRule0011ForTableFields": true
}
```
![image](https://user-images.githubusercontent.com/102963427/206154967-13e03dec-e90d-4cf7-b782-314db09cdb8f.png)

**Comment by jthoerner-tso** [comment]
In my projects the rule "enableRule0011ForTableFields" is always false positive, so that the warning is always displayed on the field level, no matter which Access type is set.
![image](https://user-images.githubusercontent.com/63199461/218124503-cffc8cb2-447b-4039-ba0c-aca9328ad618.png)


In my ruleSet.json

```
{
  "id": "LC0011",
  "action": "Warning",
  "justification": "Every object needs to specify a value for the Access property"
}
```


My LinterCop.json
```
{
    "cyclomaticComplexityThreshold": 8,
    "maintainabilityIndexThreshold": 20,
    "enableRule0011ForTableFields": true,
    "enableRule0016ForApiObjects": true
}
```

**Comment by Arthurvdv** [comment]
Hi, over the past months I’ve been putting a lot of time and care into the successor of BusinessCentral.LinterCop: ALCops (https://alcops.dev/). Because of that, this repository is no longer actively maintained.

I’m closing this issue, as it won’t be handled here anymore. If this is still relevant for you, please have a look at ALCops/Analyzers (https://github.com/ALCops/Analyzers).

To make the transition easier, you can also follow the LinterCop Migration Guide: https://alcops.dev/docs/lintercop-migration/

If you have any questions or run into issues during the transition, feel free to reach out through ALCops. Happy to help 🙂
